tastytaps/taps/urls.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import routers
+
+from . import api
+
+
+router = routers.SimpleRouter()
+
+router.register(r'brewery', api.BreweryViewSet)
+router.register(r'price', api.PriceViewSet)
+router.register(r'taps', api.TapsViewSet)
```

Comment:
I notice that 2 of these are singular and taps is plural. I wonder if we should be consistent one way or another?